src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.8.12
+-----
+
+ * [BC BREAK] Changed values of the Symfony\Component\Form\Extension\Validator\Constraints\Form constants.
```

Comment:
We cannot have a BC break in a patch release. Is it really a BC break? People MUST use the constant anyway, not the value.